third-party-docs/README-dockerhub.md

```diff
@@ -0,0 +1,35 @@
+# What is hipcheck?
+
+Hipcheck is a command line interface (CLI) tool for analyzing open source
+software packages and source repositories to understand their software supply
+chain risk. It analyzes a project's _software development practices_ and
+detects _active supply chain attacks_ to give you both a long-term and
+immediate picture of the risk from using a package.
+
+## How to use the image
```

Comment:
Nitpick: Consistent capitalization with other headers would make this "How to Use the Image"